crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -736,8 +736,9 @@ impl CheckpointStore {
         &mut self,
         checkpoint: &CertifiedCheckpointSummary,
         committee: &Committee,
+        contents: Option<&CheckpointContents>,
```

Comment:
This feels strange to me. Since we are promoting a signed checkpoint to cert, we should only need to check that the local hash matches before calling this

Reply:
You're right - removed the param. I was only ever passing None anyway.